### ISSUE TITLE
Show panel on save if errors are present?

### ISSUE THREAD

**Issue by glenjamin** [steelbrain/linter-ui-default#458]
Hello!

I'm migrating over from SublimeText + SublimeLinter, so far this linting package is great, and this UI is really neat and clean.

One of the settings I'm missing from my old setup that I think would work really well here is the option to show all errors on save - for this module i think that could be acheived by having the option to toggle the linter panel to open if there are any errors when saving the file.

Happy to have a go at contributing if you're open to the feature.

**Comment by Arcanemagus** [comment]
There's already an option to automatically show/hide the panel if there are messages, is that not what you mean here?

**Comment by glenjamin** [comment]
Not quite, the idea is that as i’m editing I rely on the inline as-you-type linting.

Then when I hit save, that’s some level of finality to what i’m doing - I’d always save before not coming back to a file.

So I want the linter to remind me at that point that there out outstanding issues - similar to a compile error workflow.

I’d then either choose to dismiss the panel because i’m still working, or work through the list to call it done.

In SublimeLinter the on-save list appears as an overlay list similar to cmd+p, which is neat as you can hit escape to close it or select an item from the list via keyboard.
